docs/scalable-functions/README.md

```diff
@@ -0,0 +1,4 @@
+Scalable function (SF) is a lightweight function that has a parent PCI function on which it is deployed. Scalable function doesn't have PCI bus of its own, but is using [Auxiliary Bus](https://www.kernel.org/doc/html/latest/driver-api/auxiliary_bus.html) instead.
+
+One of the implementation is [Mellanox/ScalableFunctions](https://github.com/Mellanox/scalablefunctions/blob/main/README.md).
+Mellanox Scalable Function has its own function capabilities and its own resources. This means a Scalable Function has its own dedicated queues(txq, rxq, cq, eq). These queues are neither shared nor stolen from the parent PCI function. There is no special support needed from system BIOS to use Mellanox Scalable Functions. Scalable Functions do not require enabling PCI SR-IOV and co-exist with PCI SR-IOV Virtual Functions.
```

Comment:
can you add a section of how to configure SFs so they can be consumed by device plugin ?

also we should have a heading for the page e.g

# Scalable Functions

add info about it

## Configure Scalable Function with Nvidia NIC

add info how to configuration with example sriov dp config to consume the resource

Reply:
> can you add a section of how to configure SFs so they can be consumed by device plugin ?

Isn't Mellanox wiki page enough? It is well described there.

> 
> also we should have a heading for the page e.g
> 
> # Scalable Functions
> add info about it
> 
> ## Configure Scalable Function with Nvidia NIC
> add info how to configuration with example sriov dp config to consume the resource

Ok